web-app/src/pages/tournament/TournamentCreator.vue

```diff
@@ -260,6 +249,13 @@ export default class TournamentCreator extends Vue {
   }
 
   private validation() {
+    if (
+      this.activeTournamentType === 'single-elimination' ||
+      this.activeTournamentType === 'round-robin'
+    )
+      return true;
+    else return false;
+
```

Comment:
I'm not sure but You  wanna check data below.?So You cant return here true. Validation fun should check all if's and return false on first not valid object. On the end of the function should be `return true` to return true when all objects are valid

Reply:
true af
correcting right away